core-curriculum/knowledge/linux.json

```diff
@@ -0,0 +1,225 @@
+            "tags":["required-for-junior"],
+            "dependencies":["basic"],
+            "abilities": [],
+            "resources":[]
+        },
+        {
+            "id": "packagemanagementadvanced",
+            "name": "Package management II",
+            "description": "At some point when working with Linux you will have to deal with software that is not available from the normal package repositories. The result of this is that you have to find a non-official source, download binaries directly and install them manually, or even build software from source. Doing this without making a mess of your system, or worse, interfering with your package manager, requires some knowledge, discipline, and skill.",
+            "tags":[],
+            "dependencies":["fsh","packagemanagement"],
+            "abilities": [],
+            "resources":[]
+        },
+        {
+            "id": "diskmanagement",
+            "name": "Managing Disks",
+            "description":"In the end the filesystem of a Linux machine is a located on a block device, there has to be at least one present mounted at the root of the filesystem. Blockdevices can be backed by pretty much anything. This topic is about block devices, how to manage and how to format them.",
+            "tags":[],
+            "dependencies":["fsh"],
+            "abilities": [],
+            "resources":[]
+        },
+        {
+            "id": "usermanagement",
+            "name": "Users and Groups",
+            "description": "Linux has users, and groups to group those users. There exist various tools for creating, listing and managing these entities. In addition to that there are several special users/groups, such as \"root\" and several methods of privilege escalation.",
+            "tags":["required-for-junior"],
+            "dependencies":["basic"],
+            "abilities": [],
+            "resources":[]
+        },
+        {
+            "id": "permissions",
+            "name": "File Access Permissions",
+            "description":"Access to files in Linux is primarily determined by the user and group who own the files and the permissions set on the file. In addition to that it is possible to add further ACLs to linux files, for example to give multiple groups access to the same directory tree. Managing file permissions is what this topic is about.",
+            "tags":[""],
+            "dependencies":["usermanagement"],
+            "abilities": [],
+            "resources":[]
+        },
+        {
+            "id": "servicemanagement",
+            "name": "Managing Services",
+            "description":"Common tools for managing services on Linux, primarily systemd.",
```

Comment:
can we turn systemd into its own topic? As it is used by pretty much every distribution right now

Reply:
I'd prefer that people learn about the principle behind service management, i.e.: some process that is responsible for making sure all services start and the right order that has to be configged and then use SystemD as a leading example.

Then we can add a follow-up topic like "advanced service management with systemd".